web/src/lib/components/faces-page/assign-face-side-panel.svelte

```diff
@@ -200,13 +165,11 @@
         </div>
       </button>
       <div class="w-full flex">
-        <input
-          class="w-full gap-2 bg-immich-bg dark:bg-immich-dark-bg"
-          type="text"
-          placeholder="Name or nickname"
-          bind:value={searchName}
-          on:input={searchPeople}
-          use:initInput
+        <SearchPeople
+          type="input"
+          bind:searchName
```

Comment:
I don't think you need to bind the name since `SearchPeople` won't update it

Reply:
Same as before, but I can probably simplify with something like that https://github.com/immich-app/immich/pull/9082/commits/15c1e89c6f1ce4425461253d04724cb76e501582